bindings/cs/MongoDB.Crypt/IKmsKeyId.cs

```diff
@@ -37,5 +48,31 @@ public interface IKmsKeyId
     internal interface IInternalKmsKeyId
     {
         void SetCredentials(ContextSafeHandle handle, Status status);
+        void SetAlternateKeyNames(ContextSafeHandle handle, Status status);
+    }
+
+    /// <summary>
+    /// Using extension methods in lieu of default interface methods that will not be available until C#8 releases
+    /// </summary>
+    internal static class KmsExtensions
```

Comment:
Should the name be `IKmsKeyIdExtensions`?

Our naming pattern has been that extension methods for type `Xyz` are in `XyzExtensions`.

Reply:
fixed